docs_2.0/manual_EN/5.deployment-and-installation/1.resource-preparations.md

```diff
@@ -28,7 +28,7 @@ For now, we can only compile Nebula Graph in the Linux system. We recommend that
 
 ### Software requirements for compiling Nebula Graph
```

Comment:
line 15: how about change the title `Requirements for compiling Nebula Graph Source Code`

Reply:
That's indeed clearer. Thank u!